### ISSUE TITLE
[BUG]: Environments assigned with Security Groups does not appear in apps

### ISSUE THREAD

**Issue by firozozman** [microsoft/powerapps-tools#321]
Environments assigned with Security Groups does not appear in apps.

Power Platform Dashboard does not display environment that has security groups assigned to it even though the current user is part of the group.
Apps and Flows from those environments also not displayed in the view.

**Component (please tell us which flow or app you are experiencing issues with):**
 - Core Solution
 - Version 1.45
 - App or Flow - Power Platform Dashboard, Developer Compliance Center, 

**To Reproduce**
1. Install and configure Core Solution and all related apps and flows
2. Set Security Group for an environment
3. Go to Power Platform Admin View > Environments
4. You will not see the environment from step 2

**Expected behavior**
All the environments that the logged in user have Read-Write access to and its flows and apps must be visible under admin view.

**Comment by JeneferM-MSFT** [comment]
Hi. I'm afraid I dont quite follow you here. Can you please expand?

2) What do you mean by step 2 "Set Security Group for an environment". Can you share a screenshot? 

3) In step 3, are you going to the product's admin portal? [https://admin.powerplatform.microsoft.com/environments](https://admin.powerplatform.microsoft.com/environments)

**Comment by firozozman** [comment]
2. Please check this link. https://docs.microsoft.com/en-us/power-platform/admin/control-user-access#associate-a-security-group-with-a-common-data-service-environment 
I have environments with Security Groups assigned to them. They dont appear in the Admin Dashboard app that came with CoE Core component.

3. Power Platform Admin View is the default app that comes with CoE Core component.

**Comment by JeneferM-MSFT** [comment]
Ah I see. Thank you. 
Is the user account that set up / runs the sync flows a member of the security group? 
The sync flows will only see and gather environment information for environments to which they have access.

**Comment by firozozman** [comment]
Yes. The account is part of the Security Group. Still cant see the environment in the Dashboard.

**Comment by JeneferM-MSFT** [comment]
And have the sync flows ran since this environment created and since you made that change?

**Comment by firozozman** [comment]
Yes they run every night successfully. I can see the environments with no security groups assigned on the Dashboard.

**Comment by JeneferM-MSFT** [comment]
ok. I dont repro so I will investigate a little more. It would be helpful to know
1) If you can repro this on multiple environments or just one
2) If you did any other changes to the envt settings besides the change you describe above.

**Comment by firozozman** [comment]
1. I don't have another tenant to try this unfortunately
2. No other settings changes besides this.

**Comment by JeneferM-MSFT** [comment]
Can you please fetch the offending environment's information by running a flow like this, and then send me the output?
![image](https://user-images.githubusercontent.com/62659043/83664988-9f48cf00-a57f-11ea-8840-ca6e5099e811.png)

**Comment by firozozman** [comment]
Sure, by the way, we are also using PIM to activate Admin roles. Unsure if it has to do with anything.

**Comment by JeneferM-MSFT** [comment]
Along with the above (information from the tenant), can you please also get me the information from the CoE for this environment? Either by opening in excel, or showing all the data in the entity view. 
![image](https://user-images.githubusercontent.com/62659043/83665606-7d038100-a580-11ea-817b-beb3e988e580.png)

**Comment by firozozman** [comment]
![Environment Entities](https://user-images.githubusercontent.com/15437706/83667177-01afc880-a59c-11ea-94ef-5953608fde41.png)

**Comment by JeneferM-MSFT** [comment]
**FROM TENANT:**
Glad you can fetch the info from Get Environment as Admin.  Can you please send me all of the outputs from the Get Environment as Admin call? 

**FROM COE:**
You may not have the Excel addin available to you, can you please get the data using the other technique I showed, go to Data > All Fields. Then show me here all of the information in the CoE about that environment.

**Comment by JeneferM-MSFT** [comment]
Ah never mind on the **"FROM COE"**, I see your screenshot above. Both of those environments show as deleted in the CoE. We filter out deleted environments in the CoE dashboards. 

When you query the tenant for the environement with _Get Environment as Admin_, you will get back these properties. Can you validate that you can browse to them? Just checking to be sure these werent deleted by someone and in the process of being cleaned yet.
![image](https://user-images.githubusercontent.com/62659043/83669745-ebe3d880-a586-11ea-8792-362169fdd662.png)

If so, you can clean up the CoE by clearing out the values in the two fields around deletion:
Environment Deleted & Environment Deleted On

Not sure how you got into this state, but this should get you back on track.
